package.json

```diff
@@ -430,12 +565,31 @@
         "when": "cmake:enableFullFeatureSet",
         "title": "%cmake-tools.command.cmake.outline.setLaunchTarget.title%"
       },
+      {
+        "command": "cmake.projectStatus.setLaunchTarget",
+        "when": "cmake:enableFullFeatureSet",
+        "title": "%cmake-tools.command.cmake.outline.setLaunchTarget.title%",
+        "icon": {
+          "dark": "res/dark/edit-icon.svg",
+          "light": "res/light/edit-icon.svg"
+        }
+      },
       {
         "command": "cmake.stop",
         "title": "%cmake-tools.command.cmake.stop.title%",
         "when": "cmake:enableFullFeatureSet",
         "category": "CMake"
       },
+      {
+        "command": "cmake.projectStatus.stop",
+        "title": "%cmake-tools.command.cmake.stop.title%",
+        "when": "cmake:enableFullFeatureSet",
+        "category": "CMake",
+        "icon": {
```

Comment:
It looks like you can use `$(x)` for this one.  (We may be able to delete most or all of the svg's in our repo since you discovered this icon syntax is supported).  The reference page for all icons is here: https://code.visualstudio.com/api/references/icons-in-labels